packages/loader/container-loader/src/loader.ts

```diff
@@ -486,7 +487,7 @@ export class Loader implements IHostLoader {
     private async loadContainer(
         request: IRequest,
         resolved: IFluidResolvedUrl,
-        pendingLocalState?: IPendingContainerState,
+        pendingLocalState?: unknown,
```

Comment:
@wes-carlson -- can you please review loader.ts and container.ts here

Reply:
https://github.com/microsoft/FluidFramework/pull/9557/files#diff-c94f4bb2aaac11f9fc45cb8bfd7c09d091d54e38e437b727ac1244f1e8d5cb6c

Reply:
I think this should be reverted:

```suggestion
        pendingLocalState?: IPendingContainerState,
```